radio/src/targets/flysky/board.h

```diff
@@ -335,12 +335,12 @@ enum Analogs {
   STICK4,
   SW_A,   
   SW_B,
-  SW_C,
-  SW_D,
   POT_FIRST,
   POT1 = POT_FIRST,
   POT2,
   POT_LAST = POT2,
+  SW_C,
```

Comment:
Why this order needs to change?

Reply:
Changed order reflects, how the channels are actually reside in `adcValues` buffer.

Reply:
It seems that it affects calibration data order for users, can you revert it or prove otherwise?

Reply:
Cannot confirm. Calibration works with changed order as well. Could you provide more details plz?

Reply:
in `menuCommonCalib` There is condition for POTS: `if (i >= POT_1 && i <= POT_LAST)` with logic to set midpoint.
If I understand it correctly, previous calibration center point will become invalid after this change.
If you need specific index, use `ana_mapping[]` array, see adc_driver for explanation.